frontend/packages/dev-console/src/components/import/__tests__/deployImage-submit-utils.spec.ts

```diff
@@ -46,4 +58,117 @@ describe('DeployImage Submit Utils', () => {
       expect(values.isi.ports).toContainEqual({ containerPort: 8081, protocol: 'TCP' });
     });
   });
+
+  describe('createDeployment tests', () => {
+    beforeAll(() => {
+      jest
+        .spyOn(k8s, 'k8sCreate')
+        .mockImplementation((model, data, dryRun) => ({ model, data, dryRun }));
```

Comment:
Why some with `Promise` and some without?

Reply:
changed